scripts/installer.sh

```
@@ -49,10 +49,10 @@ wait_for_crds() {
 }
 
 install_basic_charts() {
-  if [ "$BUNDLE" = "paas-full" ] || [ "$BUNDLE" = "distro-full" ]; then
-  make -C packages/system/cilium apply resume
+  if [ "$BUNDLE" = "paas-full" ] || [ "$BUNDLE" = "distro-full" || "$BUNDLE" = "paas-proxmox" ]; then
```

Shellcheck

[error] Instead of [ a || b ], use [ a ] || [ b ].

(SC2109)